apps/explorer/src/pages/object-address/index.tsx

```diff
@@ -0,0 +1,280 @@
+	INPUT_OBJECT = 'inputObject',
+	OUTPUT_OBJECT = 'outputObject',
+}
+
+function Header({
+	address,
+	loading,
+	error,
+}: {
+	address: string;
+	loading?: boolean;
+	error?: Error | null;
+}) {
+	const { data: domainName, isLoading, error: resolveSuinsError } = useResolveSuiNSName(address);
+	const { data, isPending, error: getObjectError } = useGetObject(address!);
+	const errorText = getObjectError?.message ?? resolveSuinsError?.message ?? error?.message;
+
+	return (
+		<div>
+			<PageHeader
+				error={errorText}
+				loading={loading || isLoading || isPending}
+				type={data ? 'Object' : 'Address'}
+				title={address}
+				subtitle={domainName}
+				before={<ObjectDetailsHeader className="h-6 w-6" />}
+				after={<TotalStaked address={address} />}
+			/>
+
+			<ErrorBoundary>
+				{data && (
+					<div className="mt-5">
+						<ObjectView data={data} />
+					</div>
+				)}
+			</ErrorBoundary>
+		</div>
+	);
+}
+
+function OwnedObjectsSection({ address }: { address: string }) {
+	const isMediumOrAbove = useBreakpoint('md');
+
+	const leftPane = {
+		panel: (
+			<div className="mb-5 h-coinsAndAssetsContainer">
+				<OwnedCoins id={address} />
+			</div>
+		),
+		minSize: LEFT_RIGHT_PANEL_MIN_SIZE,
+		defaultSize: LEFT_RIGHT_PANEL_MIN_SIZE,
+	};
+
+	const rightPane = {
+		panel: <OwnedObjects id={address} />,
+		minSize: LEFT_RIGHT_PANEL_MIN_SIZE,
+	};
+
+	return (
+		<TabHeader title="Owned Objects" noGap>
+			<div className="flex h-full flex-col justify-between">
+				<ErrorBoundary>
+					{isMediumOrAbove ? (
+						<SplitPanes
+							autoSaveId={LOCAL_STORAGE_SPLIT_PANE_KEYS.ADDRESS_VIEW_HORIZONTAL}
+							dividerSize="none"
+							splitPanels={[leftPane, rightPane]}
+							direction="horizontal"
+						/>
+					) : (
+						<>
+							{leftPane.panel}
+							<div className="my-8">
+								<Divider />
+							</div>
+							{rightPane.panel}
+						</>
+					)}
+				</ErrorBoundary>
+			</div>
+		</TabHeader>
+	);
+}
+
+function TransactionsSection({ address }: { address: string }) {
+	const client = useSuiClient();
+	const [activeTab, setActiveTab] = useState<string>(TABS_TRANSACTIONS_VALUES.ADDRESS);
+
+	const {
+		data: transactionsForAddressData,
+		isPending,
+		isError,
+		isLoading,
+	} = useQuery({
+		queryKey: ['transactions-for-address', address],
+		queryFn: () =>
+			client.queryTransactionBlocks({
+				filter: {
+					FromAndToAddress: {
+						from: address,
+						to: address,
+					},
+				},
+				order: 'descending',
+				limit: 100,
+				options: {
+					showEffects: true,
+					showInput: true,
+				},
+			}),
+	});
+
+	const renderTransactionForAddressTable =
+		isLoading || (transactionsForAddressData?.data && transactionsForAddressData.data.length > 0);
+
+	useEffect(() => {
+		if (!renderTransactionForAddressTable) {
+			setActiveTab(TABS_TRANSACTIONS_VALUES.INPUT_OBJECT);
+		}
+	}, [renderTransactionForAddressTable]);
+
+	return (
+		<Tabs size="lg" value={activeTab} onValueChange={setActiveTab}>
```

Comment:
I think the tabs UI is the wrong UI here, I think you want the inline filter UI we use today